lib/config/optionsconfiguration.go

```diff
@@ -52,7 +53,7 @@ type OptionsConfiguration struct {
 	TrafficClass            int      `xml:"trafficClass" json:"trafficClass"`
 	DefaultFolderPath       string   `xml:"defaultFolderPath" json:"defaultFolderPath" default:"~"`
 	SetLowPriority          bool     `xml:"setLowPriority" json:"setLowPriority" default:"true"`
-	MaxConcurrentScans      int      `xml:"maxConcurrentScans" json:"maxConcurrentScans"`
+	RawMaxConcurrentFolders int      `xml:"maxConcurrentFolders" json:"maxConcurrentFolders"`
```

Comment:
MaxFolderConcurrency? Current one reads badly and doesn't really explain what it does. Feels like a limit of how many folders I can have.